include/model.php

```diff
@@ -757,6 +757,21 @@ public function add_missing_secondary_language_terms() {
 		$this->clean_languages_cache();
 	}
 
+	/**
+	 * Returns the default language.
+	 *
+	 * @since 3.4
+	 *
+	 * @return PLL_Language|false Default language object, false if no language found.
+	 */
+	public function get_default_language() {
+		if ( empty( $this->options['default_lang'] ) ) {
+			return false;
```

Comment:
Shouldn't we take the opportunity to return null instead?

Reply:
Done in e9fb84b